pallets/ddc-staking/src/lib.rs

```diff
@@ -173,14 +173,14 @@ pub mod pallet {
 	pub type Ledger<T: Config> =
 		StorageMap<_, Blake2_128Concat, T::AccountId, StakingLedger<T::AccountId, BalanceOf<T>, T>>;
 
-	/// The map of (wannabe) CDN participants stash keys to the DDC cluster ID they wish to
+	/// The map of (wannable) CDN participants stash keys to the DDC cluster ID they wish to
 	/// participate into.
 	#[pallet::storage]
 	#[pallet::getter(fn cdns)]
 	pub type CDNs<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, ClusterId>;
 
-	/// The map of (wannabe) storage network participants stash keys to the DDC cluster ID they wish
-	/// to participate into.
+	/// The map of (wannable) storage network participants stash keys to the DDC cluster ID they
+	/// wish to participate into.
```

Comment:
Can you provide a better doc string here? The storage is called `Storages` but what is it a storage for?

Reply:
Yes, they are Storage nodes that participate in DDC network